src/libmongoc/tests/json/initial_dns_seedlist_discovery/load-balanced/loadBalanced-no-results.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This is not yet part of the spec, but proposed in https://github.com/mongodb/specifications/pull/1035